package.json

```diff
@@ -5,7 +5,7 @@
     "test": "npm run lint && jest",
     "test:watch": "jest --watchAll",
     "test:current": "jest --watch",
-    "prettier": "prettier --single-quote --trailing-comma=none --write 'app/javascript/**/*.js' '*.{js,json}'",
+    "prettier": "prettier --write 'app/javascript/**/*.js' '*.{js,json}'",
```

Comment:
ok i still think these are a little more readable up here, but whatever floats your boat ;) 

It does not seem that you get "single source of truth" (eslint & prettier configs) unless you are using the default prettier settings and [estlint-config-prettier](https://prettier.io/docs/en/eslint.html#turn-off-eslint-s-formatting-rules).  Since we are planning to override here, and then subsequently consume from an eslint plugin upstream, I think we'll just have to add back the eslint rule overrides later on (depending on if the upstream config goes to 120 or not).

Clear as mud 😆

Reply:
right 🤦‍♂️ 🏩 💘